modules/manually-restoring-cluster-etcd-backup.adoc

```diff
@@ -0,0 +1,312 @@
+2022-06-28T19:52:43Z    info    membership/cluster.go:421   added member    {"cluster-id": "c5996b7c11c30d6b", "local-member-id": "0", "added-peer-id": "fdc2725b3b70127c", "added-peer-peer-urls": ["https://10.0.94.214:2380"], "added-peer-is-learner": false}
+----
+.. Exit from the container.
+
+.. Repeat these steps on all the control plane hosts, checking that the values printed in the added member logs are the same.
+
+. Move the regenerated etcd database to the default location.
+
+.. Move the regenerated database (that is, the `member` folder) in the etcd default location `/var/lib/etcd`:
++
+[source,terminal]
+----
+$ sudo mv /var/lib/etcd/restore-<UUID>/member /var/lib/etcd
+----
+
+.. Restore the SELinux context for `/var/lib/etcd/member` on `/var/lib/etcd`:
++
+[source,terminal]
+----
+$ sudo restorecon -vR /var/lib/etcd/
+----
+
+.. Remove the leftovers files and directories:
++
+[source,terminal]
+----
+$ sudo rm -rf /var/lib/etcd/restore-<UUID>
+$ sudo rm /var/lib/etcd/<snapshot_yyyy-mm-dd_hhmmss>.db
+----
++
+In the end, `/var/lib/etcd` must contain only the folder `member`.
+
+.. Repeat these steps on all the control plane hosts.
+
+. Restart the etcd cluster.
+
+.. The following steps must be executed on all control plane hosts, but one host at a time.
+
+.. Move the `etcd` static pod manifest back to the kubelet manifest directory, in order to make kubelet start the related containers :
++
+[source,terminal]
+----
+$ sudo mv /tmp/etcd-pod.yaml /etc/kubernetes/manifests
+----
+
+.. Verify that all the `etcd` containers have started:
++
+[source,terminal]
+----
+crictl ps | grep etcd | grep -v operator
+----
++
+.Example output
+[source,terminal]
+----
+38c814767ad983       f79db5a8799fd2c08960ad9ee22f784b9fbe23babe008e8a3bf68323f004c840                                                         28 seconds ago       Running             etcd-health-monitor                   2                   fe4b9c3d6483c
+e1646b15207c6       9d28c15860870e85c91d0e36b45f7a6edd3da757b113ec4abb4507df88b17f06                                                         About a minute ago   Running             etcd-metrics                          0                   fe4b9c3d6483c
+08ba29b1f58a7       9d28c15860870e85c91d0e36b45f7a6edd3da757b113ec4abb4507df88b17f06                                                         About a minute ago   Running             etcd                                  0                   fe4b9c3d6483c
+2ddc9eda16f53       9d28c15860870e85c91d0e36b45f7a6edd3da757b113ec4abb4507df88b17f06                                                         About a minute ago   Running             etcdctl    
+----
++
+If the output of the following command is empty, wait a few minutes and check again
+
+. Check the status of the etcd cluster.
```

Comment:
I would be consistent with how you are using [backticks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) with etcd. For example, in line 239 and 268 you aren't using them, but then you use them in line 270. I did a quick search of the repo and see that etcd cluster has been used without markup.

Reply:
For this I actually followed this particular resource, https://access.redhat.com/node/6965021/moderation/diff/view/23130153/23284287. Wherever the backtick was provided, I added them in here as well.

Reply:
And the it appears to be a step kind of heading here so I did not use backticks for etcd.

Reply:
Steps and headings are different. Although you are correct if this is a heading it would not get markup. I suppose my larger point is just make sure that you are using backticks consistently.

Reply:
Yes, I will follow the same.